README.md

```diff
@@ -51,15 +51,25 @@ To check you've set it up correctly, run `docker-compose --version` and it shoul
 
 2. Navigate to the directory: `cd student-net`
 
-3. Start the Docker containers
+3. Create a .env file instead of the `cd web` directory.
```

Comment:
Can you add a `.env.example` file as well?

Reply:
We have *.env set in .gitignore, so I made a sample file in the readme